### ISSUE TITLE
Add support for default graph

### ISSUE THREAD

**Issue by chrhansen** [somaticio/tensorflow.rb#69]
__Retrieving Default Graph__
It should be possible to retrieve a default graph [See Python doc](https://www.tensorflow.org/versions/r0.10/api_docs/python/framework.html#get_default_graph). This might be the first thing to build as the code for creating ops and running a session would use the "get-default-graph" method.

__Creating Ops__
Unless explicitly specified nodes should be added to this default graph:
```ruby
Tensorflow.constant([1.0, 2.0, 3.0], shape: [3], name : 'a')
# or short-hand:
TF.constant([1.0, 2.0, 3.0], shape: [3], name : 'a')
```

__Session__
Also, when `Session.run` is called it should grab the default graph if not explictly specified
https://www.tensorflow.org/versions/r0.10/api_docs/python/client.html#Session.run

Python `Graph` documentation: https://www.tensorflow.org/versions/r0.10/api_docs/python/framework.html#Graph

**Comment by chrhansen** [comment]
@geoffreylitt didn't you mention at some point you wanted to look into adding support for default graph?

**Comment by geoffreylitt** [comment]
Yep! Earlier discussion was here: https://github.com/somaticio/tensorflow.rb/pull/52

Last couple weeks have been busy and I haven't had time to make progress on an implementation, but I was planning to work on this in the next week or so.
